src/bin/common.rs

```diff
@@ -82,12 +84,16 @@ pub fn parse_cli() -> CliOptions {
     )
     // ENCODING SETTINGS
     .arg(
-      Arg::with_name("PASS")
-        .help("Specify first-pass or second-pass to run as a two-pass encode; If not provided, will run a one-pass encode")
-        .short("p")
-        .long("pass")
+      Arg::with_name("FIRST_PASS")
```

Comment:
Maybe call this FIRST_PASS_FILE for more clarity in --help?

Reply:
I thought it was silly to add new options in the first place instead of using `--stats` and `--pass`... Perhaps ask Tim?

Reply:
Yeah that makes sense. The previous --stats option actually outputs a "human readable" json file rather than the machine-readable first pass file... we could rename that one to something else.

I think potentially the idea of the two options is to allow x many passes, by both having a pass input and output file at the same time.

Reply:
I would suggest there's no reason not to have them be the same JSON file. The intent behind the JSON output file is that someone could write a script to extract data they cared about from the first-pass file (e.g. keyframe numbers). I don't think it would conflict with that goal to have additional fields in that file, and having just one stats file would be simpler IMO.

Reply:
I'm strongly against having a JSON based stats file, especially when we introduce temportal RDO, which has a *lot* of data...

Reply:
We already use bincode for the machine-readable file for the train-rdo, we can use it for the pass files.

Reply:
I do not have a strong opinion. What works well for you works for me :)